README.md

```diff
@@ -1,24 +1,25 @@
-## Daedalus Dock codebase
+# Dionysus Dock codebase
 
-[![CI Suite](https://github.com/DaedalusDock/daedalusdock/actions/workflows/ci_suite.yml/badge.svg)](https://github.com/DaedalusDock/daedalusdock/actions/workflows/ci_suite.yml)
+<!-- [![CI Suite](https://github.com/DaedalusDock/daedalusdock/actions/workflows/ci_suite.yml/badge.svg)](https://github.com/DaedalusDock/daedalusdock/actions/workflows/ci_suite.yml)
 [![Percentage of issues still open](https://isitmaintained.com/badge/open/DaedalusDock/daedalusdock.svg)](https://isitmaintained.com/project/DaedalusDock/daedalusdock "Percentage of issues still open")
 [![Average time to resolve an issue](https://isitmaintained.com/badge/resolution/DaedalusDock/daedalusdock.svg)](https://isitmaintained.com/project/DaedalusDock/daedalusdock "Average time to resolve an issue")
-![Coverage](https://img.shields.io/badge/coverage-no%25-red.svg)
+![Coverage](https://img.shields.io/badge/coverage-no%25-red.svg) -->
 
 [![resentment](.github/images/badges/built-with-resentment.svg)](.github/images/comics/131-bug-free.png) [![technical debt](.github/images/badges/contains-technical-debt.svg)](.github/images/comics/106-tech-debt-modified.png) [![forinfinityandbyond](.github/images/badges/made-in-byond.gif)](https://www.reddit.com/r/SS13/comments/5oplxp/what_is_the_main_problem_with_byond_as_an_engine/dclbu1a)
 
-* **Website:** https://daedalus13.net
+<!-- * **Website:** https://daedalus13.net
 * **Code:** https://github.com/DaedalusDock/daedalusdock
 * **Wiki:** https://wiki.daedalus13.net/Main_Page
 * **Codedocs:** https://codedoc.daedalus13.net/
 * **Daedalus Dock Discord:** https://discord.gg/bttwJmfznH
-* **Coderbus Discord:** https://discord.gg/Vh8TJp9
+* **Coderbus Discord:** https://discord.gg/Vh8TJp9 -->
```

Comment:
If the discord link is still valid for CBD, it can stay. The rest will have to be done in time; the wiki and such will be done at a later date once we're closer to going public.